### DISCUSSION TITLE
MVT line layer

### DISCUSSION THREAD

**Discussion by imakihi** [TerriaJS/terriajs#5083] [Q&A] [answered]
Hi there.
I have been trying to show a road line MVT layer with Terria V8 but I couldn't make it work. I could show polygons without an issue. I made sure tiles were coming to my browser but nothing showed up.

<img width="545" alt="2020-12-14_9-27-30" src="https://user-images.githubusercontent.com/4824436/102114087-abdf9700-3dee-11eb-9747-0d1a9586f24a.png">

I could see the line data using QGIS like below.
<img width="227" alt="2020-12-14_9-30-11" src="https://user-images.githubusercontent.com/4824436/102114473-17c1ff80-3def-11eb-96e5-2d803fc1aa95.png">

Can anybody tell me that I can render MVT line geometry with Terria V8? Did I do something wrong? Here is my catalog description.

	{
          "name": "road",
          "type": "mvt",
		  "layer": "traffic_layer_hl",
          "url": "https://.....com/traffic/tiles/timeline/2020-10-16/{z}/{x}/{y}.mvt"
        },

and one tile sample.

[400.zip](https://github.com/TerriaJS/terriajs/files/5690557/400.zip)

Thank you.

Hiroo

**Comment by geospatialmapmashup** [comment]
lineColor: 'hsl(56,100%,50%,1)'
you are missing some required parameters you need to provide the styling information

**Reply by imakihi** [reply]
@geospatialmapmashup Thank you for your quick response!
I actually tried the option too but I couldn't show my lines. I just added your line to my JSON like following.

`	{
          "name": "road",
          "type": "mvt",
		  "layer": "traffic_layer_hl",
          "url": "https://.....com/traffic/tiles/timeline/2020-10-16/{z}/{x}/{y}.mvt",
		  "lineColor": 'hsl(56,100%,50%,1)'
        },`

and I got nothing...

<img width="1222" alt="2020-12-14_10-10-12" src="https://user-images.githubusercontent.com/4824436/102118708-eea46d80-3df4-11eb-9d47-d4e36ea51435.png">

What do you think?

Hiroo

**Comment by zoran995** [comment, marked as answer]
@imakihi Terria currently supports only mvt polygons, check the #3846 and #4633

**Reply by imakihi** [reply]
@zoran995 Thank you for the clarification. Hopefully I can use points and lines with MVT sometime.
